threadbare/operations.py

```diff
@@ -120,8 +162,9 @@ def _execute(command, user, key_filename, host_string, port, use_pty):
     keep this function as simple as possible."""
 
     # https://parallel-ssh.readthedocs.io/en/latest/native_single.html#pssh.clients.native.single.SSHClient
-    password = None # we *never* use passwords, not even for bootstrapping. always private keys.
-    client = SSHClient(host_string, user, password, port, pkey=key_filename)
+    #password = None # we *never* use passwords, not even for bootstrapping. always private keys.
```

Comment:
~we *never* use passwords,, not even for bootstrapping. always private keys.~
"always use private keys"